_posts/en/2024-10-09-wallet-integration-guide.md

```diff
@@ -0,0 +1,600 @@
+
+bitcoin-cli -regtest -rpcwallet=test submitpackage '["030000000001013b64742e658ff39da0288a5010f094a9e94e6ce3a3ce2ed84b74babc017aea490000000000fdffffff01f90295000000000016001400991cdadccdf30cb5a04663b0371cb433a095b40247304402200a82f2fd8aa5f32cdfd9540209ccfc36a95eea21518ede1c3787561c8fb7269702207a258e6f027ce156271879c38628ad9b3425b83c33d8cd95fb20dd3c567fdff70121030af1fadce80bcb8ba614634bc82c71eea2ed87a5692d3127766cc896cef1bdb100000000", "03000000000101de7615100656cdc2f8fd0217fbbae0d7c6cea020c7d9f64ada16d269db6491bf0000000000fdffffff0100ed94000000000016001400991cdadccdf30cb5a04663b0371cb433a095b4024730440220685a6d76db97b2c27950f267b70d606f1864002ff6b4617cd2e29afd5ddfac83022037be8bb2ebe8194b4263f16a634e5c00a5f6c4eef0968d12994ed66dcf15b9ac0121020797cc343a24dfe49c7ee9b94bf3daaf15308d8c12e3f0f7e102b95ee55f939f00000000"]'
+{
+  "package_msg": "success",
+  "tx-results": {
+    "7d855ffbd8bc17892e28f3f326d0e4919d35c27a7370f5d9f9ce538e93a347cf": {
+      "txid": "bf9164db69d216da4af6d9c720a0cec6d7e0bafb1702fdf8c2cd5606101576de",
+      "vsize": 110,
+      "fees": {
+        "base": 0.00000000,
+        "effective-feerate": 0.00025568,
+        "effective-includes": [
+          "7d855ffbd8bc17892e28f3f326d0e4919d35c27a7370f5d9f9ce538e93a347cf",
+          "4333b3d2eea820373262c7ffb768028bc82f99f47839349722eb60c58cd65b55"
+        ]
+      }
+    },
+    "4333b3d2eea820373262c7ffb768028bc82f99f47839349722eb60c58cd65b55": {
+      "txid": "6c2f4dec614c138703f33e6a5c215112bad4cf79593e9757105e09b09bf3e2de",
+      "vsize": 110,
+      "fees": {
+        "base": 0.00005625,
+        "effective-feerate": 0.00025568,
+        "effective-includes": [
+          "7d855ffbd8bc17892e28f3f326d0e4919d35c27a7370f5d9f9ce538e93a347cf",
+          "4333b3d2eea820373262c7ffb768028bc82f99f47839349722eb60c58cd65b55"
+        ]
+      }
+    }
+  },
+  "replaced-transactions": [
+  ]
+}
+```
+
+The 1P1C package has entered into the local mempool at an effective feerate of
+25.568 sats per vB even though the parent transaction is below minrelay feerate.
+Success!
+
+## TRUC Transactions
+
+Topologically Restricted Until Confirmation (TRUC) transactions, previously known as v3
+transactions, is a new, opt-in, [mempool policy][policy series] aimed at allowing robust
+replace-by-fee (RBF) of transactions, mitigating both fee-related transaction
+[pinning][topic transaction pinning] as well as package limit pinning. Its central philosophy is: while many
+features are infeasible for all transactions, we can implement them for packages
+with a limited topology, so we create a way to opt-in to this more robust set of
+policies in addition to the topological restrictions.
+
+In short, a TRUC transaction is a transaction with an nVersion of 3, which
+restricts the transaction to either a singleton of up to 10kvB, or the child of
+exactly one TRUC transaction capped at 1kvB. Non-TRUC and TRUC transactions cannot
+be spent together unconfirmed in the mempool. All TRUC transactions are
```

Comment:
This sentence kind of confused me. Spent together, as in share a child? But we already know they can't have multiple parents. It's also not covering the exact rule. Maybe

```suggestion
exactly one TRUC transaction capped at 1kvB. A TRUC transaction cannot spend a non-TRUC transaction, and vice versa. All TRUC transactions are
```